testing/run_api_tests.sh

```diff
@@ -37,6 +36,33 @@ fi
 
 # Initialise test data
 case $1 in
+    postgres|mysql|mariadb)
+    docker compose -f "$script_dir/docker-compose.yml" down
+    docker volume rm "testing_wakapi-$1"
+
+    docker_down=1
+    docker compose -f "$script_dir/docker-compose.yml" up --wait -d "$1"
+    if [ "$1" == "mariadb" ]; then
+        config="config.mysql.yml"
+    else
+        config="config.$1.yml"
+    fi
+
+    db_port=0
+    if [ "$1" == "postgres" ]; then
+        db_port=5432
```

Comment:
I'd probably use some different, less common port (e.g. `55432` or so). This way, tests could still run, even if a Postgres database (/ MySQL, respectively) is already running on the machine for a different purpose.